src/workerd/io/trace-stream.c++

```diff
@@ -58,7 +58,6 @@ namespace {
   V(OK, "ok")                                                                                      \
   V(ONSET, "onset")                                                                                \
   V(OUTCOME, "outcome")                                                                            \
-  V(PARENTSPANID, "parentSpanId")                                                                  \
```

Comment:
Why?

Reply:
`parentSpanId` and `trigger` are being removed based on the updated API – I believe the discussion at https://github.com/cloudflare/workerd/pull/4727/files#r2261753832 addresses the concerns raised about this.